canonical/charmed-kubeflow.io
# fix(page): remove /tutorials #6259

## Done

- Removed /tutorials section
- Added a 410 status template page to indicate the page was removed

## QA

- Visit https://charmed-kubeflow-io-141.demos.haus/tutorials
- See that you're presented with a 410 status page
- Search the codebase for "tutorials", it shouldn't appear anywhere besides the deleted.yaml

## Issue / Card

Fixes https://github.com/canonical/web-squad/issues/6259
